src-docs/src/views/guidelines/writing_guidelines.js

```diff
@@ -132,70 +133,119 @@ export const WritingGuidelines = () => {
         <EuiTitle>
           <h2 id="capitalization">Capitalization</h2>
         </EuiTitle>
+        <EuiText>
+          <p>
+            In sentence case, only the first word and proper names are capped.
+            In title case, the first letter of each word, except for articles,
+            conjuctions, and prepositions, is capitalized.
+          </p>
+        </EuiText>
 
-        <GuideRule
-          heading="Sentence case for almost all text"
-          description="This includes buttons, menus, and titles. In sentence case, only the
-    first word and proper names are capped."
-        >
+        <GuideRule heading="Sentence case for UI elements" description=" ">
           <GuideRuleExample
             type="do"
-            text="Sentence case makes titles easier to read."
+            text="Use sentence case in buttons, menus, titles, and tabs."
+            minHeight={200}
           >
-            <EuiTitle size="xs">
-              <span>Create index patterns</span>
-            </EuiTitle>
+            <EuiButton fill>Check for new data</EuiButton>
           </GuideRuleExample>
-
           <GuideRuleExample
-            type="dont"
-            text="Title case can feel more cluttered."
+            type="do"
+            text="Use sentence case for modal and page elements."
+            minHeight={200}
           >
-            <EuiTitle size="xs">
-              <span>Create Index Patterns</span>
-            </EuiTitle>
+            <EuiPanel style={{ transform: 'scale(.75)' }}>
+              <EuiTitle size="m">
+                <span>Search session complete</span>
+              </EuiTitle>
+              <EuiSpacer />
+              <EuiText>
+                <p>Save your session and return to it later.</p>
+              </EuiText>
+              <EuiSpacer />
+              <EuiFlexGroup justifyContent="flexEnd" gutterSize="none">
+                <EuiButtonEmpty>Manage sessions</EuiButtonEmpty>
+                <EuiButton fill size="s">
+                  Save session
+                </EuiButton>
+              </EuiFlexGroup>
+            </EuiPanel>
           </GuideRuleExample>
         </GuideRule>
 
-        <GuideRule heading="" description="">
+        <GuideRule heading="Sentence case for feature names" description=" ">
```

Comment:
```suggestion
        <GuideRule heading="Sentence case for feature names">
```
No need to include the `description` prop if it's empty!